packages/design-system/src/components/Button/style.scss

```diff
@@ -90,6 +90,28 @@
     }
   }
 
+  &--text {
+    background-color: transparent;
+    color: $font-primary-blue-color;
+    border-color: transparent;
+    font-weight: 400;
+    padding: 6px 3px;
+
+    &:hover {
+      color: $hover-font-primary-blue-color;
+      background-color: transparent;
+    }
+
+    &.btn .btn__loader {
+      justify-content: flex-start;
+    }
+
+    &.btn--large, &.btn--compact {
+      padding-left: 3px;
```

Comment:
hmm, usually larger button has bigger paddings: 
lc-btn--compact - `padding: 4px 16px;
lc-btn - `padding: padding: 6px 16px;`
lc-btn--large - `padding: 9px 24px;`

Shouldn't it be the same?

Reply:
I'll discuss it Jo.

Reply:
Discussed and fixed in cb726c9 - text buttons will behave like other buttons :-)